plugins/hls-semantic-tokens-plugin/test/testdata/T1.expected

```diff
@@ -0,0 +1,81 @@
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 30, _startChar = 4, _len = 1}, _name = "x"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 30, _startChar = 9, _len = 1}, _name = "c"}
+SemanticTokenOriginal {_tokenType = TFunction, _loc = Loc {_line = 30, _startChar = 14, _len = 2}, _name = "aa"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 30, _startChar = 17, _len = 1}, _name = "x"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 30, _startChar = 19, _len = 1}, _name = "c"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 31, _startChar = 12, _len = 2}, _name = "xx"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 31, _startChar = 16, _len = 2}, _name = "yy"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 32, _startChar = 11, _len = 2}, _name = "dd"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 34, _startChar = 2, _len = 2}, _name = "zz"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 34, _startChar = 6, _len = 2}, _name = "kk"}
+SemanticTokenOriginal {_tokenType = TFunction, _loc = Loc {_line = 35, _startChar = 1, _len = 2}, _name = "cc"}
+SemanticTokenOriginal {_tokenType = TTypeCon, _loc = Loc {_line = 35, _startChar = 7, _len = 3}, _name = "Foo"}
+SemanticTokenOriginal {_tokenType = TTypeCon, _loc = Loc {_line = 35, _startChar = 15, _len = 3}, _name = "Int"}
+SemanticTokenOriginal {_tokenType = TTypeCon, _loc = Loc {_line = 35, _startChar = 20, _len = 3}, _name = "Int"}
+SemanticTokenOriginal {_tokenType = TTypeCon, _loc = Loc {_line = 35, _startChar = 28, _len = 3}, _name = "Int"}
+SemanticTokenOriginal {_tokenType = TFunction, _loc = Loc {_line = 36, _startChar = 1, _len = 2}, _name = "cc"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 36, _startChar = 4, _len = 1}, _name = "f"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 36, _startChar = 7, _len = 2}, _name = "gg"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 36, _startChar = 11, _len = 2}, _name = "vv"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 37, _startChar = 10, _len = 2}, _name = "gg"}
+SemanticTokenOriginal {_tokenType = TRecField, _loc = Loc {_line = 38, _startChar = 14, _len = 3}, _name = "foo"}
+SemanticTokenOriginal {_tokenType = TFunction, _loc = Loc {_line = 38, _startChar = 18, _len = 1}, _name = "$"}
+SemanticTokenOriginal {_tokenType = TVariable, _loc = Loc {_line = 38, _startChar = 20, _len = 1}, _name = "f"}
+SemanticTokenOriginal {_tokenType = TRecField, _loc = Loc {_line = 38, _startChar = 24, _len = 3}, _name = "foo"}
+SemanticTokenOriginal {_tokenType = TRecField, _loc = Loc {_line = 39, _startChar = 14, _len = 3}, _name = "foo"}
+SemanticTokenOriginal {_tokenType = TFunction, _loc = Loc {_line = 39, _startChar = 18, _len = 1}, _name = "$"}
```

Comment:
Hmm, maybe we should be using "operator" for infix things? But then it'll be awkward - what about infix class methods, are they methods or operators? Worth a comment at least.